encfs/CipherFileIO.cpp

```diff
@@ -127,6 +154,49 @@ bool CipherFileIO::setIV(uint64_t iv) {
   return base->setIV(iv);
 }
 
+/**
+ * Helper function to adjust plain size to cipher size
+ * according to configuration (file header, padding).
+ */
+void CipherFileIO::plainSizeToCipherSize(off_t *size) const {
```

Comment:
It's good that it is now a separate function. This means it can get a unit test. I think it needs one!